Ginger/Ginger/Actions/ActionEditPages/ValidationDBPage.xaml.cs

```diff
@@ -266,7 +266,10 @@ private void DBNameComboBox_SelectionChanged(object sender, SelectionChangedEven
                     if (ValidationCfgComboBox.Items.Cast<ComboEnumItem>().Where(m => m.text.ToString().Equals(eDBValidationType.Insert.ToString())) != null
                         && ValidationCfgComboBox.Items.Cast<ComboEnumItem>().Where(m => m.text.ToString().Equals(eDBValidationType.Insert.ToString())).Count() != 0)
                     {
-                        ValidationCfgComboBox.SelectedIndex = 0;
+                        if (ValidationCfgComboBox.SelectedValue.ToString() == eDBValidationType.Insert.ToString())
```

Comment:
Add null check for selected value as well before checking for insert value

Reply:
Done